pkg/messagebroker/models.go

```diff
@@ -30,6 +34,7 @@ type CommitOnTopicRequest struct {
 	Topic     string
 	Partition int32
 	Offset    int64
+	ID        string
```

Comment:
This message ID right? So, commit on kafka is per offset, but for pulsar per message? Can it be offset based in Pulsar too? If not, have separate methods?